.github/workflows/publishProjectFactory.yml

```
@@ -1,77 +1,72 @@
-name: project factory service docker Image CI
+name: Project Factory Service Test Builder
 
 on:
```

yamllint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)